src/core/renderer.ts

```diff
@@ -45,18 +45,7 @@ export abstract class Renderer<E extends Element, S extends Snapshot<E> = Snapsh
   }
 
   createScriptElement(element: Element) {
-    if (element.getAttribute("data-turbo-eval") == "false") {
-      return element
-    } else {
-      const createdScriptElement = document.createElement("script")
-      if (this.cspNonce) {
-        createdScriptElement.nonce = this.cspNonce
-      }
-      createdScriptElement.textContent = element.textContent
-      createdScriptElement.async = false
-      copyElementAttributes(createdScriptElement, element)
-      return createdScriptElement
-    }
+    return createScriptElement(element)
```

Comment:
It seems to me that this will create a loop. On line 83 you redefine a function with the same name.

I think that adding `export` on line 47 would have been enough.

Reply:
no, this was done on purpose, it's `return createScriptElement(element)`, not `return this.createScriptElement(element)` 

I took the `createScriptElement` function out of the `abstract class Renderer`'s scope, so that I could import the function with a simple `import { createScriptElement } from "../renderer";` in `stream_message.ts`. And at the same time I left `createScriptElement` in the `Renderer` so that it wouldn't break other code that still use Renderer#createScriptElement

A loop won't be created because on line 48 I'm calling the line 83's `createScriptElement`.

If I return everything as it was and just add `export` on line 47, I won't be able to import the function `createScriptElement` with `import { createScriptElement } from "../renderer";` in `stream_message.ts`, because I would have to implement and create an object of `abstract class Renderer`, which I'm quite frankly unable to do due to my incompetence

I assure you, my solution works, the tests are passing. I wouldn't push the code and waste your time if tests were failing.

Reply:
Hi @jean-francois-labbe ! How are things your end?
Can you please approve this PR?

Reply:
Hi @elShiaLabeouf things are great.
I cannot approve this PR I'm not a maintainer on Hotwired.

Reply:
Oh. Sorry to bother then. Thank you for the hints on the code improvements!